docs/_autosummary/toqito.matrix_ops.inner_product.rst

```diff
@@ -1,6 +0,0 @@
-toqito.matrix\_ops.inner_product
```

Comment:
Hmm, I'm not sure we want to delete the files that are already present here. 

I think you are correct though in expecting that these *should* be updated as part of the docs build when new features go in. However, I think these ones (which are presently on `master` shouldn't be removed.

Reply:
Sure. I deleted those because `make html` was editing them. 

![image](https://github.com/vprusso/toqito/assets/66048318/ab8f905b-f554-4ed9-96e5-64ba8a07af5f)

I will copy-paste what's in the current files.

Reply:
Sure, you could probably also just do `git checkout <NAME-OF-FILE>` as well to replace them with what is presently on `master`. Either way works!